lib/ipaddress/ipv4.rb

```diff
@@ -805,6 +805,42 @@ def subnet(subprefix)
       end
     end
 
+    #
+    # Subtract other from the current range
+    # Returns list of ranges that excludes :other
+    #
+    # Example:
+    #
+    #   a = IPAddress '10.0.0.0/16'
+    #   b = IPAddress '10.0.128.0/18'
+    #   c = IPAddress '192.168.0.0/16'
```

Comment:
Can we come up with better names than a, b, c?

Reply:
yes. But I don't that other test cases follow this..